front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -301,10 +301,10 @@ export default function AssistantBuilder({
 
   // This state stores the slack channels that should have the current agent as default.
   const [selectedSlackChannels, setSelectedSlackChannels] = useState<
-    {
-      channelId: string;
-      channelName: string;
-    }[]
+    | {
```

Comment:
What is the `|` doing here ?

Reply:
this shouldn't do anything 🤨

Reply:
ah need to clean-up will do in subsequent PR

Reply:
https://github.com/dust-tt/dust/commit/723cfb2fc09a95fd5cac59c8bf603cd3fc35982e